tools/yaml-templates/build-test-publish.yml

```diff
@@ -118,10 +118,10 @@ steps:
 
   - task: CopyFiles@2
     inputs:
-      Contents: |
-        packages\teams-js\dist\**\?(*.js|*.ts|*.map)
+      sourceFolder: 'packages\teams-js\dist'
```

Comment:
would this include the packages in `umd`?

Reply:
Yes, it will include both the umd and esm packages. We still need to publish both the umd package to npm to backwards support any apps that are not configured to use es modules